src/components/SplashScreen.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <v-dialog v-model="showDialog" max-width="900" @click:outside="showDialog = false">
```

Comment:
This should be an accept/consent button (on the terms and conditions), not click outside.